src/current/cockroachcloud/production-checklist.md

```diff
@@ -6,9 +6,63 @@ cloud: true
 docs_area: deploy
 ---
 
-This page provides important recommendations for CockroachDB {{ site.data.products.cloud }} production deployments.
+Before deploying CockroachDB {{ site.data.products.cloud }} in production, it is important to understand the Shared Responsibility Model that delineates the responsibilities of Cockroach Labs and the customer in managing CockroachDB {{ site.data.products.cloud }} clusters. 
 
-## Follow the SQL Best Practices
+Under the Shared Responsibility Model, Cockroach Labs is responsible for the following tasks: 
+
+- Cluster and cloud service availability and reliability
+- Maintenance and security of hardware and operating systems
+- Database and security patches 
+- Cluster backups 
+
+The customer is responsible for the following tasks:
+
+- Estimating workload and sizing the cluster 
+- Scaling clusters based on workload
+- Ensuring sufficient disk, compute, and memory capacity for each cluster
+- Monitoring cluster health and application performance
+- Ensuring that the workload is distributed appropriately across the nodes of the cluster
+- Performance tuning of SQL queries and schema
+- Initiating major version upgrades and selecting maintenance windows for patch releases 
+- (Optional) Taking customer-owned backups
+
+This page provides important recommendations for CockroachDB {{ site.data.products.cloud }} production tasks for which  the customer is responsible.
+
+## Deployment options
+
+When planning your deployment, it is important to carefully review and choose the [deployment options](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/choose-a-deployment-option) that best meet your scale, cost, security, and resiliency requirements. 
+
+Make sure your cluster has sufficient storage, CPU, and memory to handle the workload. The general formula to calculate the storage requirement is as follows:
+ 
+`raw data (storage, in GB) * replication factor (3 by default) * remove 40% to account for compression (0.6) * headroom (1.5-2)`
+
+For an example, see [Plan your Dedicated cluster]({% link cockroachcloud/plan-your-cluster.md %}).
+
+## Topology patterns
+
+When planning your deployment, it is important to carefully review and choose the [topology patterns](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/topology-patterns) that best meet your latency and resiliency requirements. This is especially crucial for multi-region deployments.
+
+## Cluster management
+
+You can create and manage CockroachDB {{ site.data.products.cloud }} clusters using the [Cloud Console](http://cockroachlabs.cloud), [Cloud API]({% link cockroachcloud/cloud-api.md %}), [ccloud CLI]({% link cockroachcloud/ccloud-get-started.md %}), or the [Terraform provider]({% link cockroachcloud/provision-a-cluster-with-terraform.md %}).
```

Comment:
Is it worth making a recommendation to choose a configuration-as-code approach, since we are in the context of making recommendations?

Reply:
The intent here was to list the options available to the customer so they won't have to figure them out on their own. Not sure if we want to prescribe which option to use 🤔